src/client/dfs/dfs.c

```diff
@@ -2292,6 +2295,15 @@ dfs_remove(dfs_t *dfs, dfs_obj_t *parent, const char *name, bool force,
 	return rc;
 }
 
+int
+dfs_remove(dfs_t *dfs, dfs_obj_t *parent, const char *name, bool force, daos_obj_id_t *oid)
+{
+	if (oid && (oid->lo || oid->hi))
```

Comment:
why? one could pass in an oid to get back, but values is garbage.
you impose that being also set to 0 now..

Reply:
Agreed, the alternative would be to just set oid to 0 here if non-zero which would be more backwards compatible.